conda_build/main_develop.py

```diff
@@ -42,6 +43,21 @@ def main():
                    help=("Relink compiled extension dependencies against "
                          "libraries found in current conda env. "
                          "Do not add source to conda.pth."))
+    p.add_argument(
+                   '-b', '--build_ext',
+                   action='store_true',
+                   help=("Build extensions inplace, invoking: "
+                         "python setup.py build_ext --inplace; "
+                         "add to conda.pth; relink runtime libraries to "
+                         "environment's lib/."))
+    p.add_argument(
+                   '-c', '--clean_build_ext',
```

Comment:
Looking at the code that should be simple, no? You just need to split [this](https://github.com/conda/conda-build/pull/512/files#diff-8ebb521eb386d609915cfbd151033d13R176) block into a separate function. 

It's only a few extra characters to type `conda develop --clean --build_ext` vs. `conda develop --clean_build_ext` and it seems much cleaner (no pun intended).